CHANGES.md

```diff
@@ -1,5 +1,13 @@
 # Change Log
 
+### 1.116 - 2024-04-01
+
+#### @cesium/engine
+
+##### Fixes :wrench:
```

Comment:
We typically list these deprecation under `##### Breaking Changes :mega:`
```suggestion
##### Breaking Changes :mega:
```

Reply:
I fixed it